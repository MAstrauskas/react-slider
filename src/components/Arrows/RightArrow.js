import React from "react";
import Arrow from "../../assets/icons/right-arrow.svg";

import "./Arrows.css";

const RightArrow = (props) => {
    return (
            <div className="arrow" onClick={props.handleNextSlide}>
                <img src={Arrow} alt="Right Arrow" aria-hidden="true"/>
            </div>
        )
};

export default RightArrow;