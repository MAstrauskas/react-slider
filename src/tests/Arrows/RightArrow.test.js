import React from "react";
import { create } from "react-test-renderer";
import RightArrow from "../../components/Arrows/RightArrow";

describe("Right Arrow Component", () => {
    test("it matches the snapshot", () => {
        const component = create(<RightArrow/>);
        expect(component.toJSON()).toMatchSnapshot();
    })
});