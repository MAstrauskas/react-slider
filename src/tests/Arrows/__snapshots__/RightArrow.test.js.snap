// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Right Arrow Component it matches the snapshot 1`] = `
<div
  className="arrow"
>
  <img
    alt="Right Arrow"
    aria-hidden="true"
    src="right-arrow.svg"
  />
</div>
`;
